addons/google_calendar/data/google_calendar_data.xml

```diff
@@ -14,7 +14,7 @@
 model._sync_all_google_calendar()
             </field>
             <field name="user_id" ref="base.user_root" />
-            <field name="interval_number">12</field>
+            <field name="interval_number">2</field>
```

Comment:
Not too sure about changing this default, as it could cause a lot of unnecessary processing, e.g. on the saas. Unless we want to add logic to speed up the cron *only* when someone configures a working sync, I would just leave it at 12 by default, and let admins manually change it when they need it.

Reply:
you're right and since the synchro is trigger anyway each time you call /google_calendar/sync_data, more less each time you load the calendar view